WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -33,7 +33,7 @@ def show
     @ranks_average = @person.ranksAverage
     @medals = @person.medals
     @records = @person.records
-    @results = @person.results.includes(:competition, :event, :format, :round_type).order("Events.rank, Competitions.start_date DESC, RoundTypes.rank DESC")
+    @results = @person.results.includes(:competition, :event, :format, :round_type).order("Events.rank, Competitions.id, Competitions.start_date DESC, RoundTypes.rank DESC")
```

Comment:
I think we want `id` *after* `start_date` here!

Reply:
Oh dear! Sorry ;_;